1.1/servicedescription/servicedescription_all_functions.ttl

```diff
@@ -0,0 +1,68 @@
+@prefix sd: <http://www.w3.org/ns/sparql-service-description#> .
+@prefix geo: <http://www.opengis.net/ont/geosparql#> .
+@prefix geof: <http://www.opengis.net/def/function/geosparql/> .
+@prefix ent: <http://www.w3.org/ns/entailment/> .
+@prefix prof: <http://www.w3.org/ns/owl-profile/> .
+@prefix void: <http://rdfs.org/ns/void#> .
+
+[] a sd:Service ;
+    sd:endpoint <http://www.example/sparql/> ;
+    sd:supportedLanguage sd:SPARQL11Query ;
+	sd:extensionFunction geof:area;
+	sd:extensionFunction geof:asDGGS;
+	sd:extensionFunction geof:asKML;
+	sd:extensionFunction geof:asGeoJSON;
+	sd:extensionFunction geof:asGML;
+	sd:extensionFunction geof:asWKT;
+	sd:extensionFunction geof:boundary;
+	sd:extensionFunction geof:buffer;
+	sd:extensionFunction geof:convexHull;
+	sd:extensionFunction geof:difference;
+	sd:extensionFunction geof:dimension;
+	sd:extensionFunction geof:distance;
+	sd:extensionFunction geof:ehContains;
+	sd:extensionFunction geof:ehCoveredBy;
+	sd:extensionFunction geof:ehCovers;
+	sd:extensionFunction geof:ehDisjoint;
+	sd:extensionFunction geof:ehEquals;
+	sd:extensionFunction geof:ehInside;
+	sd:extensionFunction geof:ehMeet;
+	sd:extensionFunction geof:ehOverlap;
+	sd:extensionFunction geof:envelope;
+	sd:extensionFunction geof:getSRID;
+	sd:extensionFunction geof:geometryN;
+	sd:extensionFunction geof:intersection;
+	sd:extensionFunction geof:isEmpty;
+	sd:extensionFunction geof:isSimple;
+	sd:extensionFunction geof:minx;
+	sd:extensionFunction geof:miny;
+	sd:extensionFunction geof:minz;
+	sd:extensionFunction geof:maxx;
+	sd:extensionFunction geof:maxy;
+	sd:extensionFunction geof:maxz;
+	sd:extensionFunction geof:numGeometries;
+	sd:extensionFunction geof:transform;
+	sd:extensionFunction geof:rcc8dc;
+	sd:extensionFunction geof:rcc8ec;
+	sd:extensionFunction geof:rcc8eq;
+	sd:extensionFunction geof:rcc8ntpp;
+	sd:extensionFunction geof:rcc8ntppi;
+	sd:extensionFunction geof:rcc8po;
+	sd:extensionFunction geof:rcc8tpp;
+	sd:extensionFunction geof:rcc8tppi;
+	sd:extensionFunction geof:relate;
+	sd:extensionFunction geof:sfContains;
+	sd:extensionFunction geof:sfCrosses;
+	sd:extensionFunction geof:sfDisjoint;
+	sd:extensionFunction geof:sfEquals;
+	sd:extensionFunction geof:sfIntersects;
+	sd:extensionFunction geof:sfOverlaps;
+	sd:extensionFunction geof:sfTouches;
+	sd:extensionFunction geof:sfWithin;
+	sd:extensionFunction geof:symDifference;
+	sd:extensionFunction geof:union;
+    sd:resultFormat <http://www.w3.org/ns/formats/RDF_XML>, <http://www.w3.org/ns/formats/Turtle> ;
+    sd:defaultEntailmentRegime ent:RDFS
+.
```

Comment:
As far as I can tell not all functions are listed here: metricBuffer, coordinateDimension, metricDistance, geometryType, is3D, isMeasured, spatialDimension, lenght, projectTo, and the aggregate functions are missing.

Also, minX, maxX, etc should be with a capital last letter.

Reply:
That is correct. Something I spotted in the other pull request which closes 9 issues.
Not sure I entered them all there, but at least some of them.
So it would be better to merge that one in first

Reply:
projectTo was removed from the specification recently. I will add the others now

Reply:
Should be changed now. Thanks for checking